internal/clients/nightfall/nightfall.go

```diff
@@ -39,6 +48,7 @@ type Client struct {
 	APIKey             string
 	Detectors          []*nightfallAPI.Detector
 	MaxNumberRoutines  int
+	InitialDelay       time.Duration
```

Comment:
added this field here so it could be configurable in our tests (instead of having to wait 30+ seconds on the max retries failure test)